src/carriers/portmonitor_carrier/lua/MonitorLua.cpp

```diff
@@ -15,7 +15,6 @@
 using namespace yarp::os;
 using namespace std;
 
-
```

Comment:
This change is useless ;)
Since this is the only change to this file I suggest to remove it, to keep the patch clean.